libs/sdk-backend-tiger/src/convertors/fromBackend/analyticalDashboards/AnalyticalDashboardConverter.ts

```diff
@@ -43,7 +43,7 @@ export const convertAnalyticalDashboard = (
     analyticalDashboard: JsonApiAnalyticalDashboardOutWithLinks,
     included?: JsonApiAnalyticalDashboardOutIncludes[],
 ): IListedDashboard => {
-    const { id, links, attributes = {}, meta, relationships = {} } = analyticalDashboard;
+    const { id, links, attributes, meta, relationships = {} } = analyticalDashboard;
```

Comment:
Why do we need to remove the default assignment? (applies to other convertors as well)

Reply:
Somebody made these properties required on backend (unrelated to this feature), so the default are not applicable to the types now.